python_scripts/cross_validation_train_test.py

```diff
@@ -168,7 +168,19 @@
 # - train a new model on the train set;
 # - evaluate the testing error on the test set.
 #
-# We repeat this procedure `n_splits` times. Using `n_splits=40` means that we
+# We repeat this procedure `n_splits` times. Keep in mind that the computational
+# cost increases with `n_splits`.
+#
+# ![Cross-validation diagram](../figures/shufflesplit_diagram.png)
```

Comment:
Here, we as regenerate with a small legend as well. But it could be done in another PR together with the other cross-validation.

Reply:
I think it's ok because the colors are explained in the body of the text, but a legend in the figure always helps.

Reply:
I will do that in a separate PR